app/src/components/Forms/AddContribution/index.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Button from "../../Button/Button";
+import AddContributionForm from "./AddContributionForm";
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+// import Portal from '../../../Pages/Portal/Portal';
+
+const formTitle = css`
+  font-size: 35px;
+  letter-spacing: -2px;
+  margin: 10px 0px;
+`;
+
+const AddContribution = () => (
+    <AddContributionForm
+      onSubmit={x => console.log("REPLACE ME WITH SOMETHING REAL!")}
+      initialValues={{
+        dateOfContribution: '07/04/2019', // KELLY - needs to be a date and validated
+        typeOfContribution: "Contribution",
+        subTypeOfContribution: "Cash Contribution",
+        amountOfContribution: 100.00,
+        oaeContributionType: "Seed Money",
+        paymentMethod: "Cash",
+        checkNumber: 192
+      }}
+    >
+      {({
+        formSections,
+        isValid,
+        handleSubmit /* isDirty, isSubmitting */
+      }) => (
+        <React.Fragment>
+        <Button
+          buttonType="submit"
+          disabled={!isValid}
+          onClick={handleSubmit}
+          >
+          Submit Contribution
+        </Button>
+          <p css={formTitle}>Basics</p>
+          <div>{formSections.dateOfContribution}</div>
+
+          <div>{formSections.typeOfContribution}</div>
+ 
+          {formSections.subTypeOfContribution} 
+          {formSections.typeOfContributor} 
+          {formSections.amountOfContribution} 
+          {formSections.oaeContributionType} 
+          {formSections.paymentMethod} 
+          {formSections.checkNumber} 
```

Comment:
I was imagining that the formSections would be a larger section, e.g. all of the above would be in `formSections.basic`. Then could be wrapped in a `div` that handles the layout of all of the fields within that section.

Reply:
👍

Reply:
@jaronheard right now I have the  `header`, `basic`, `contributor`, and `other details` sections broken out into separate forms that are all pulled together to create the AddContribution page. I did it that way so I could break it all apart in my mind as I was trying understand the structure of the code, but one flaw as it sits right now is the `submit` button is probably only aware of the `header` form where it lives.  😬 

So, before I do any more that may be outside of what you're describing here, maybe we can talk tomorrow.  I'd also love to chat with you or maybe @andrewbiang888 about some styling questions.